src/app-layer-htp.c

```diff
@@ -1755,9 +1755,7 @@ static int HtpResponseBodyHandle(HtpState *hstate, HtpTxUserData *htud,
      * we check for tx->response_line in case of junk
      * interpreted as body before response line
      */
-    if (!(htud->tcflags & HTP_FILENAME_SET) &&
```

Comment:
these are unrelated changes?

Reply:
This was related to the change @catenacyber made: https://github.com/OISF/libhtp/commit/c629f266a47600f238fa10609c111e5b73831e2a

Reply:
This PR showed up a bug in libhtp which got fixed

The http-evader tests are based on files (check for eicar) with rules

As this PR changes file detection, the file needs to be opened (instead of just getting the response body)

This may be a separate commit, and maybe a S-V test to test for file info event instead of one with rules...

Reply:
Ok thanks. Yes lets get these in through a separate commit (and PR ?)

Reply:
@jlucovsky should I do it or do you ? (the separate PR/commit for these changes)

Reply:
@catenacyber Yes -- thanks

Reply:
https://github.com/OISF/suricata/pull/9171